chrome/fed-modules.json

```
@@ -392,7 +392,7 @@
                 "id": "ansible-automation-analytics",
                 "module": "./RootApp",
                 "routes": [
-                    "/ansible/insights",
+                    "/ansible/automation-analytics",
```

No need to remove the `insights` route (it would actually cause errors in current setup)
```suggestion
                    "/ansible/automation-analytics",
                    "/ansible/insights",
```